### ISSUE TITLE
End up with .git in my home directory

### ISSUE THREAD

**Issue by stuaxo** [jbernard/dotfiles#78]
I end up with a .git symlinked to `Dotfiles/.git` this isn't a feature I'm keen on and want to keep the repo separate from the home directory.

Would you consider excluding .git when symlinking by default ?

dotfiles version is v0.6.4

**Comment by jbernard** [comment]
On Apr 27, Stuart Axon ***@***.***> wrote:
>I end up with a .git symlinked to `Dotfiles/.git` this isn't a feature 
>I'm keen on and want to keep the repo separate from the home directory.
>
>Would you consider excluding .git when symlinking by default ?

Hi Stuart, that's definitely not supposed to happen.  I don't see this 
behaviour here, are you able to give some steps to reproduce this?

-- 
Jon

**Comment by stuaxo** [comment]
That's a good question,  I haven't quite worked it out yet, just that I managed it multiple times.

Sorry - not a great bug report yet!

I am still using the version from pip, so I should probably swap to master I guess anyway ?

**Comment by stuaxo** [comment]
For now I'll close this until I can actually reproduce it... I thought I'd give you a headsup even though I'm not quite sure how it happened, since I've done it a few times, just not sure what I've been doing.

**Comment by jbernard** [comment]
On Apr 28, Stuart Axon ***@***.***> wrote:
>For now I'll close this until I can actually reproduce it... I thought 
>I'd give you a headsup even though I'm not quite sure how it happened, 
>since I've done it a few times, just not sure what I've been doing.

Ok, I haven't looked at this code in a very long time, but maybe this 
can be a reminder to make some updates.

-- 
Jon

**Comment by stuaxo** [comment]
In the past I've made bugs my own code when I'm joining paths and some presumption isn't right (e.g. subtract some path and add another).

My setup -

Dotfiles is a symlink to ~/projects/mine/dotfiles

What I was doing

In the home directory, the Dotfiles directory and at least one other directory:

Ran

dotfiles -c
and
dotfiles -s


(Mostly because I couldn't remember which flag was which).

I have .git in my list of files to ignore.
